projects/vdk-plugins/vdk-sqlite/src/vdk/plugin/sqlite/ingest_to_sqlite.py

```diff
@@ -198,12 +202,16 @@ def __create_table_if_not_exists(
     @staticmethod
     def __create_table(cur: Cursor, destination_table: str, columns: Dict[str, str]):
         """
-        Creates table from give list of columns and table name
+        Creates table from given list of columns and table name
         """
-        columns_as_sql_expression = ",".join(
-            [f"{col_name} {col_type}" for col_name, col_type in columns.items()]
-        )
-        sql = f"CREATE TABLE IF NOT EXISTS {destination_table} ( {columns_as_sql_expression} )"
+        names = [
+            f"{col_name} {col_type}"
```

Comment:
why not just quote everything?

Reply:
https://www.sqlite.org/draft/quirks.html -> Point 8

> The SQL standard requires double-quotes around identifiers and single-quotes around string literals. For example:
"this is a legal SQL column name"
'this is an SQL string literal'
SQLite accepts both of the above. But, in an effort to be compatible with MySQL 3.x (which was one of the most widely used RDBMSes when SQLite was first being designed) SQLite will also interpret a double-quotes string as string literal if it does not match any valid identifier.

I'd like to avoid this behaviour on the off chance we get it somehow. It would be extremely hard to debug.